src/get-html.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @type {Array<string>} */
+const excludedTags = ['amp-analytics', 'amp-experiment', 'amp-bind-state'];
+
+/**
+ * Returns content of HTML node
+ * @param {!string} selector
+ * @param {!Array<string>} attrs
```

Comment:
Please document. Any chance we could document a super set of allowed attributes?